docs/source/how-to/deploy-dask-clusters.rst

```diff
@@ -34,18 +52,16 @@ scheduler you must set up a ``Client``
 
 Note that the newer ``dask.distributed`` scheduler is often preferable, even on
 single workstations.  It contains many diagnostics and features not found in
-the older single-machine scheduler.  The following pages explain in more detail
-how to set up Dask on a variety of local and distributed hardware.
+the older single-machine scheduler.
 
-.. raw:: html
+.. figure:: ../images/dask-cluster-manager.svg
```

Comment:
The figure uses phrases like "Resource Manager" and "Cluster Manager" which are not used in the text. I think it would be worth introducing those in a sentence or two above

Reply:
Added some explanatory text! Let me know what you think (and if it is correct). I also linked to some "Dask notes" blog posts-- not sure if we want to do this in the docs?